src/pages/CertificatesViewAll.js

```diff
@@ -110,3 +122,31 @@ const Main = styled.div`
     max-width: 1200px;
   }
 `
+
+const PaddedDiv = styled.div`
+  display: flex;
+  flex-wrap: wrap;
+  gap: 10px;
+  padding: 34px 21px;
+`
```

Comment:
I don't think there's any need for this `div`? You can move `gap` and `padding` up to `Sidebar` and remove and it'll be the same just with less `div` soup 😋